<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertViewerForm</class>
 <widget class="QDialog" name="AlertViewerForm">
  <property name="windowTitle">
   <string>Просмотр тревожных сообщений</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTableWidget" name="alertTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="buttonAcknowledge">
       <property name="text"><string>Подтвердить</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text"><string>Закрыть</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>